### PR TITLE
Show solutions and results tabs on discussions even when no solution present

### PR DESCRIPTION
**Pull request by julian-berbel**
## :dart: Goal
Stop hiding solution and results tabs when discussion has no submission.

## :camera_flash: Screenshots
https://user-images.githubusercontent.com/11720274/132385417-31caeee7-3ff4-47f6-a0f3-1cd9864020fe.mp4

## :warning: Dependencies
None.

## :back: Backwards compatibility
Yes.

## :soon: Future work
None.

### CODE REVIEW

**Comment by julian-berbel** [comment]
@rgonzalezt gave their OK on this.

File: app/views/exercises/_read_only.html.erb [hunk 1]
```diff
@@ -77,32 +73,30 @@
...
-                <% end %>
-              </div>
+                </div>
+              <% end %>
             </div>
-          <% end %>
+          </div>
 
-          <div role="tabpanel" class="tab-pane <%= 'active' unless @discussion.has_submission? %>" id="content">
+          <div role="tabpanel" class="tab-pane" id="content">
```
Comment (felipecalvo) [inline app/views/exercises/_read_only.html.erb:99]: I'm a bit confused as to why were we adding a second `active` class for the `content` tab (the same happens with the `nav-link` further up), but this way is better as there should only be one active tab-pane at the time. I'm guessing the second one would override the first, but still, it was kinda messy...

Before merging, would you mind checking if all tabs fade consistently? Only the active tab-pane should have `show active`, but _all_ of them should have `fade` for the fade effect - or at least that's what the documentation and my quick testing suggest. I believe it should be on lines 76, 86 and 99 but please check it yourself.

See the examples here: https://getbootstrap.com/docs/5.0/components/navs-tabs/#javascript-behavior

**Review by felipecalvo — APPROVED** [review]
Great! There's a single comment on a pre-existing inconsistency regarding the discussion tabs.

File: app/views/exercises/_read_only.html.erb (hunk 1 above)
Comment (julian-berbel) [reply app/views/exercises/_read_only.html.erb]: @felipecalvo I will see your frontend mumbo jumbo and raise you properly fading tabs!
Peek 2021-09-09 14-14

**Pull request merged**